scripts/README.md

```diff
@@ -0,0 +1,21 @@
+<h2> General-purpose scripts for Reactome's release </h2>
+
+This repository contains scripts that are used by multiple release steps.
+
+
+<h3>updateDatabase.sh</h3>
```

Comment:
Is `updateDatabase.sh` the best name for it? I would expect that something with `update` in the name will update the data in an existing database. A drop-&-replace operation is _not_ what I expect from something named "updated". for this kind of script, I'd probably name it `replaceDatabase.sh`.

Reply:
Mind you, if it could also _create_ a database, then maybe `replaceDatabase.sh` is also not the best name. Worst case: `createOrReplace.sh`, but if you can think of something better that's fine too.

Reply:
Yea I liked createOrReplace, so I changed the file names to that -- I'll update the README accordingly